configs/uprm-civic.nrel-op.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "server": {
+        "connectUrl": "https://uprm-civic-openpath.nrel.gov/api/",
+        "aggregate_call_auth": "user_only"
+    },
+    "intro": {
+        "program_or_study": "program",
+        "start_month": "11",
+        "start_year": "2022",
+        "program_admin_contact": "For questions about the program, please contact Daniel Rodríguez Román at the University of Puerto Rico at Mayagüez (daniel.rodriguez6@upr.edu)",
```

Comment:
please compare against other programs, I think we need to have a `mode_of_interest` defined for programs.

Reply:
I had removed the `mode_of_interest` because I did not see in the MOU a specified mode of interest. Should I just set the mode of interest to be `e-bike` like the others?